### PR TITLE
Layout

### PR DESCRIPTION
**Pull request by Djurek74**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,10 +3,70 @@
...
+      <div class ='container-1'>
+
+      <div class ='box-1'>
+        <h3>Input</h3>
+        <div class ='btn'>
+        <Button class = 'b1'>Add</Button>
+      </div>
+      </div>
+
+      <div class='Box 2'>
```
Comment (RobertShaw1) [inline index.html]: Double check your css class on this one.

File: index.html [hunk 2]
```diff
@@ -3,10 +3,70 @@
...
+        <h3>List</h3>
+        <div class = 'btn'>
```
Comment (RobertShaw1) [inline index.html]: Hard to tell what the intent is for this div container. IMO, we're too early in the development for adding the edit and delete buttons.  I think you will realize how you want to use the edit and delete buttons as you start building out the features. Feel free to leave them for now.

**Comment by Djurek74** [comment]
Test

**Review by RobertShaw1 — COMMENTED** [review]
Good first pass.

**Pull request merged**